code/modules/projectiles/guns/beam_rifle.dm

```diff
@@ -472,7 +472,7 @@
 		if(original && (original.layer>=2.75) || ismob(original))
```

Comment:
According to a comment in projectiles.dm, it's the projectiles hit layer threshhold and stuff below isn't supposed to be hit.